scripts/generate-java-code-coverage.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+JACOCO_VERSION=0.8.12
+
+if [ "$(basename "$(pwd)")" != "java" ]; then
+    echo "This script must be run from the 'java' directory."
+    exit 1
+fi
+
+# make -C ../cpp srcs
+# make
+
+if [ ! -f org.jacoco.cli-nodeps.jar ]; then
+    wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.cli/$JACOCO_VERSION/org.jacoco.cli-$JACOCO_VERSION-nodeps.jar -O org.jacoco.cli-nodeps.jar
+fi
+
+if [ ! -f org.jacoco.agent-runtime.jar ]; then
+    wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.agent/$JACOCO_VERSION/org.jacoco.agent-$JACOCO_VERSION-runtime.jar -O org.jacoco.agent-runtime.jar
+fi
+
+python3 allTests.py --all --workers=4 --debug --jacoco $(pwd)/org.jacoco.agent-runtime.jar
+
+# We don't want any coverage for IceGridGUI
+find src/IceGridGUI -name '*.java' -exec rm {} \;
```

Comment:
Could we remove the class files like we do for the generated code?
They should all live in `src/IceGridGUI/build/classes`.

So maybe it'd be easier to just `rm -rf src/IceGridGUI/build` instead of using `find ... -exec rm`.

Reply:
Yes, this is a bug. I was attempting to remove the class files.